landing page/js/main.js

```diff
@@ -0,0 +1,74 @@
+
+function splittext() {
+	TweenMax.to('#main_info_cont', 1, 
+	{
+		opacity: 0,
+		delay: 2.5
+	}
+	)
+	;
+	TweenMax.to('#coming_soon', 0.5, 
+	{
+		opacity: 1,
+		delay: 2.5
+	}
+	)
+	;
+	var i = -10;
+	var interval = setInterval(
+		function() {
+			i++;
+			var obj = '#' + i; 
+			console.log(obj);
+			$(obj).addClass('transformation');
+			obj = '#' + (i - 1);
+			$(obj).removeClass('transformation');
+			if(i == 20){
+				clearInterval(interval);
+				start();
+			}
+		}
+		,250
+	);	
+}
+function start() {
+	$('body').css('background', '#ff2b06');
+	$('#splash').addClass('hidden');
+	TweenMax.to('#coming_soon', 0.5, 
+	{
+		opacity: 0
+	}
+	)
+	;
+	TweenMax.to('#main_info_cont', 1, 
+	{
```

Comment:
Follow standard conventions all over your code. Use format similar to call at [Line54](https://github.com/gawdsnitkkr/tedxnitkurukshetra-frontend/blob/da50cd9b4cd87d423144fb8c4828fa3da3ede4bc/landing%20page/js/main.js#L54-L61).